crates/bevy_winit/src/lib.rs

```diff
@@ -274,7 +274,7 @@ pub fn winit_runner(mut app: App) -> AppExit {
     let event_loop = app
         .world_mut()
         .remove_non_send_resource::<EventLoop<UserEvent>>()
-        .unwrap();
+        .expect("value found that was not previously present");
```

Comment:
```suggestion
        .expect("Couldn't find the EventLoop in the given app");
```

Reply:
changed error message to "no event loop for user events found in the provided app"